src/data/localStorage.ts

```diff
@@ -21,7 +21,14 @@ export function saveUsernameToLocalStorage(id: string, username: string) {
 
 export function restoreUsernameFromLocalStorage(id: string): string | null {
   try {
-    const data = localStorage.getItem(`${LOCAL_STORAGE_KEY_COLLAB}-${id}`);
+    // Keep backwards compatibility for those that have already set a username
+    // This will be safe to remove after a couple of days/weeks
+    let data = localStorage.getItem(`${LOCAL_STORAGE_KEY_COLLAB}-${id}`);
```

Comment:
```suggestion
    let data = localStorage.getItem(LOCAL_STORAGE_KEY_COLLAB);
```

Reply:
Nop, that's for backward compatibility.

Reply:
The feature was running for a few hours.. on saturday! :)